Specs/Scene/ExpressionSpec.js

```diff
@@ -847,6 +847,111 @@ defineSuite([
+    });
+
+    it('throws if sin function takes an invalid number of arguments', function() {
+        expect(function() {
+            return new Expression('sin()');
+        }).toThrowDeveloperError();
+
+        expect(function() {
+            return new Expression('sin(1, 2)');
+        }).toThrowDeveloperError();
+    });
+
+    it('evaluates tan function', function() {
+        var expression = new Expression('tan(0)');
+        expect(expression.evaluate(undefined)).toEqual(0);
+    });
+
+    it('throws if tan function takes an invalid number of arguments', function() {
+        expect(function() {
+            return new Expression('tan()');
+        }).toThrowDeveloperError();
+
+        expect(function() {
+            return new Expression('tan(1, 2)');
+        }).toThrowDeveloperError();
+    });
+
+    it('evaluates acos function', function() {
+        var expression = new Expression('acos(1)');
+        expect(expression.evaluate(undefined)).toEqual(0);
+    });
+
+    it('throws if acos function takes an invalid number of arguments', function() {
+        expect(function() {
+            return new Expression('acos()');
+        }).toThrowDeveloperError();
+
+        expect(function() {
+            return new Expression('acos(1, 2)');
+        }).toThrowDeveloperError();
+    });
+
+    it('evaluates asin function', function() {
+        var expression = new Expression('asin(0)');
+        expect(expression.evaluate(undefined)).toEqual(0);
+    });
+
+    it('throws if asin function takes an invalid number of arguments', function() {
+        expect(function() {
+            return new Expression('asin()');
+        }).toThrowDeveloperError();
+
+        expect(function() {
+            return new Expression('asin(1, 2)');
+        }).toThrowDeveloperError();
+    });
+
+    t('evaluates atan function', function() {
+        var expression = new Expression('atan(0)');
+        expect(expression.evaluate(undefined)).toEqual(0);
+    });
+
+    it('throws if atan function takes an invalid number of arguments', function() {
+        expect(function() {
+            return new Expression('atan()');
+        }).toThrowDeveloperError();
+
+        expect(function() {
+            return new Expression('atan(1, 2)');
+        }).toThrowDeveloperError();
+    });
+
+    it('evaluates radians function', function() {
+        var expression = new Expression('radians(0)');
+        expect(expression.evaluate(undefined)).toEqual(0);
+    });
+
+    it('throws if radians function takes an invalid number of arguments', function() {
+        expect(function() {
+            return new Expression('radians()');
+        }).toThrowDeveloperError();
+
+        expect(function() {
+            return new Expression('radians(1, 2)');
+        }).toThrowDeveloperError();
+    });
+
+    it('evaluates degrees function', function() {
+        var expression = new Expression('degrees(0)');
+        expect(expression.evaluate(undefined)).toEqual(0);
```

Comment:
Also tweak this test to something like `degrees(360)`, `toEqual(Math.PI)`. If floating precision is an issue, you can use `toEqualEpsilon`.